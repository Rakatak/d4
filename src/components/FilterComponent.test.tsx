import renderer from 'react-test-renderer';
import { FilterComponent } from './FilterComponent';

describe('Test for DataTable', () => {
  it('Renders the data table component and tests the snapshot', () => {
    const component = renderer.create(
      <FilterComponent/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

