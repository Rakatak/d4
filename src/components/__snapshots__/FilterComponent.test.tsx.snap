// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for DataTable Renders the data table component and tests the snapshot 1`] = `
Array [
  <input
    aria-label="Search Input"
    className="sc-bczRLJ fMDoqu"
    id="search"
    placeholder="Filter By Name"
    type="text"
  />,
  <button
    className="sc-gsnTZi gCvjue"
    type="button"
  >
    X
  </button>,
]
`;
